### ISSUE TITLE
Question about tuning set

### ISSUE THREAD

**Issue by yakazimir** [princeton-nlp/SimPO#74]
Did you use a special validation set for ultrafeedback when tuning the hyper-paramaters in Table 7, or just the `test_pref` set from the [original binarized ultrafeedback data](https://huggingface.co/datasets/HuggingFaceH4/ultrafeedback_binarized). I notice that in your on-policy datasets you only include `train` and `test` splits (and that the test splits are exactly the `test_pref` instances).
